### PR TITLE
pihole: update more information link

### PR DESCRIPTION
**Pull request by Iksas**
Updates the more information link and replaces `pihole updatePihole` by the equivalent and more well-known `pihole -up`.

- [ ] The page (if new), does not already exist in the repository.
- [x] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [x] The page has 8 or fewer examples.
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [x] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The page description includes a link to documentation or a homepage (if applicable).

**Version of the command being documented (if known):** v5.10

### CODE REVIEW

File: pages/linux/pihole.md [hunk 1]
```diff
@@ -1,15 +1,15 @@
...
 > Terminal interface for the Pi-hole ad-blocking DNS server.
-> More information: <https://pi-hole.net>.
+> More information: <https://docs.pi-hole.net/core/pihole-command/>.
 
 - Check the Pi-hole daemon's status:
 
 `pihole status`
 
-- Update Pi-hole:
+- Update Pi-hole (software and adlists):
```
Comment (CleanMachine1) [inline pages/linux/pihole.md:10]:
```suggestion
- Update Pi-hole and Gravity:
```

File: pages/linux/pihole.md [hunk 2]
```diff
@@ -1,15 +1,15 @@
...
 
-`pihole updatePihole`
+`pihole -up`
```
Comment (CleanMachine1) [inline pages/linux/pihole.md:12]:
```suggestion
`pihole -up`
```

I was the one to add this example to this page, and even I agree that you should update like this, the Pi-Hole arguments are weirdly named but using `-up` is better.

File: pages/linux/pihole.md (hunk 1 above)
Comment (Iksas) [reply pages/linux/pihole.md]: Those are the words I was looking for, thanks for this suggestion!

File: pages/linux/pihole.md (hunk 2 above)
Comment (Iksas) [reply pages/linux/pihole.md:12]: Yeah, I had never even heard of `updatePihole` before I read the tldr.

**Review by CleanMachine1 — APPROVED** [review]

**Review by blueskyson — APPROVED** [review]

**Pull request merged**